src/routes/blog/post/developer-tools-appwrite/+page.markdoc

```diff
@@ -0,0 +1,136 @@
+---
+layout: post
+title: 12 developer tools to supercharge your Appwrite project
+description: Boost your developer experience with Appwrite using these awesome tools.
+date: 2024-06-17
+cover: /images/blog/developer-tools-appwrite/cover.png
+timeToRead: 5
+author: aditya-oberai
+category: open-source
+---
+
+Any developer-focused product is only as good as the ecosystem of developer tools surrounding it. Fortunately, over the years, we have seen several outstanding tools developed by our community, as well as a few we have developed in-house to make development with Appwrite far more productive.
+
+# Our favourite developer tools for Appwrite
+
+## ngx-appwrite
+
+ngx-appwrite is a library designed to integrate Appwrite services seamlessly into Angular 16+ applications. It acts as a wrapper around the Appwrite Web SDK, simplifying the use of Appwrite's features within Angular projects. The library enhances developer productivity by providing RxJS streams, which facilitate reactive programming patterns commonly used in Angular applications. This integration allows developers to easily implement Appwrite's backend services, such as database, authentication, and storage while leveraging Angular's powerful framework for building dynamic web applications.
+{% cards %}
+{% cards_item href="https://www.npmjs.com/package/ngx-appwrite" title="NPM package" %}
+{% /cards_item %}
+{% cards_item href="https://github.com/blackfan23/ngx-appwrite" title="GitHub repo" %}
+{% /cards_item %}
+{% /cards %}
+
+## Auth UI
+
+Auth UI offers a customizable login flow for applications, featuring a user interface that can be adapted to different authentication methods in Appwrite. The customizable UI allows developers to implement login mechanisms that align with their specific application requirements, providing flexibility in user management and security configurations. This project demonstrates the versatility of Appwrite in supporting diverse authentication needs across various projects.
+{% cards %}
+{% cards_item href="https://www.authui.site/" title="Live application" %}
+{% /cards_item %}
+{% cards_item href="https://builtwith.appwrite.io/projects/6467cedd4502d0e29205/" title="Built With Appwrite page" %}
+{% /cards_item %}
+{% /cards %}
+
+## adapter-appwrite
+
+adapter-appwrite is a tool designed for SvelteKit that enables developers to deploy SvelteKit applications as Appwrite functions. By using adapter-appwrite, developers can leverage SvelteKit's modern web development features, such as server-side rendering and static site generation, while utilizing Appwrite Cloud's infrastructure for their backend functionalities.
+{% cards %}
+{% cards_item href="https://www.npmjs.com/package/adapter-appwrite" title="NPM package" %}
+{% /cards_item %}
+{% cards_item href="https://github.com/lukehagar/sveltekit-adapters" title="GitHub repo" %}
+{% /cards_item %}
+{% /cards %}
```

Comment:
Using cards for these doesn't look very nice because there seems to be more vertical padding/spacing at the bottom than the top:

![image](https://github.com/appwrite/website/assets/1477010/4f35b34e-5e42-427c-ae74-dc04774eebc6).

Perhaps we can just link from some text in the paragraph? Or if there are multiple, list the links.

Reply:
I'll convert to a list of links 👍